src/event_loop.rs

```diff
@@ -175,16 +179,21 @@ impl Default for ControlFlow {
 }
 
 impl EventLoop<()> {
-    /// Alias for `EventLoopBuilder::new().build()`.
+    /// Alias for `EventLoopBuilder::new().build().unwrap()`.
     #[inline]
     pub fn new() -> EventLoop<()> {
+        EventLoopBuilder::new().build().unwrap()
+    }
+
+    /// Alias for `EventLoopBuilder::new().build()`.
+    pub fn new_result() -> Result<EventLoop<()>, CreationError> {
         EventLoopBuilder::new().build()
     }
 }
 
 impl<T> EventLoop<T> {
     #[deprecated = "Use `EventLoopBuilder::<T>::with_user_event().build()` instead."]
-    pub fn with_user_event() -> EventLoop<T> {
+    pub fn with_user_event() -> Result<EventLoop<T>, CreationError> {
```

Comment:
This is deprecated, so I don't think it makes sense to change the signature; rather just make it panic

Reply:
Made it panic now